packages/blocs/authentication/lib/bloc/authentication_bloc.dart

```diff
@@ -15,11 +15,9 @@ class AuthenticationBloc
     extends Bloc<AuthenticationEvent, AuthenticationState> {
   @override
   AuthenticationState get initialState => Uninitialized();
-  final Auth _auth;
+  final Auth auth;
```

Comment:
It is done to make work with data a little faster and dont have unneeded objects in widget tree.